tests/test_multi_program.py

```diff
@@ -18,8 +19,7 @@ def sig_prog(proc, sig):
     if not proc.poll():
         # TODO: why sometimes does SIGINT not work on teardown?
         # seems to happen only when trace logging enabled?
-        proc.send_signal(signal.SIGKILL)
-
+        proc.send_signal(signal.SIGKILL if platform.system() != 'Windows' else signal.CTRL_BREAK_EVENT)
```

Comment:
This line might hit the 79 char limit I was loosely sticking to.
Not sure if there's any others.
If you think that's too stingy maybe I'll just not stick to it :thinking: